openstudiocore/src/energyplus/Test/ScheduleInterval_GTest.cpp

```diff
@@ -144,9 +147,112 @@ TEST_F(EnergyPlusFixture,ForwardTranslator_ScheduleFixedInterval_Hourly)
   EXPECT_EQ(8760, numUntils);
 }
 
+TEST_F(EnergyPlusFixture, ForwardTranslator_ScheduleFixedInterval_Hourly_Shifted)
+{
+  Vector values = linspace(1, 8760, 8760);
+
+  TimeSeries timeseries(DateTime(Date(MonthOfYear::Jan, 1)), Time(0, 1, 0), values, "");
+
+  Model model;
+
+  boost::optional<ScheduleInterval> scheduleInterval = ScheduleInterval::fromTimeSeries(timeseries, model);
+  ASSERT_TRUE(scheduleInterval);
+  EXPECT_TRUE(scheduleInterval->optionalCast<ScheduleFixedInterval>());
+
+  TimeSeries ts = scheduleInterval->timeSeries();
+  EXPECT_EQ(DateTime(Date(MonthOfYear::Jan, 1)), ts.firstReportDateTime());

```

Comment:
Shouldn't this be:

    EXPECT_EQ(DateTime(Date(MonthOfYear::Jan, 1), Time(0,1,0)), ts.firstReportDateTime());

The first interval starts at time 0 but ends at interval length, right?
https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/src/utilities/data/TimeSeries.hpp#L157